qds_sdk/nezha.py

```diff
@@ -0,0 +1,184 @@
+        NezhaCmdLine._update_parsers(update, nezha_entity)
+        
+
+        #Delete
+        delete = subparsers.add_parser("delete", help="Delete a %s"%nezha_entity)
+        delete.add_argument("id", help="Numeric id of the nezha_entity to be deleted")
+        delete.set_defaults(func=NezhaCmdLine.delete)
+
+        return argparser
+
+    @staticmethod
+    def _update_parsers(update, nezha_entity):
+        if nezha_entity == "data_sources":
+            update.add_argument("id", help="Numeric id of the DataSource")
+            update.add_argument("--url", dest="url", help="URL of the data source")
+            update.add_argument("--type", dest="type", help="Type of data source")
+            update.add_argument("--name", dest="name", help="Name of the data source")
+
+            update.add_argument("--user", dest="user", help="User name of the data source")
+            update.add_argument("--password", dest="password", help="Password of the data source")
+
+            update.add_argument("--auth_token", dest="auth_token", help="User name of the data source")
+            update.add_argument("--dbtap_id", dest="dbtap_id", help="Password of the data source")
+
+        elif nezha_entity == "cubes":
+            update.add_argument("id", help="Numeric id of the Cube")
+            update.add_argument("--cost", dest="cost", help="Cost for cubes")
+            update.add_argument("--description", dest="description", help="Description")
+            update.add_argument("--name", dest="name", help="Name of the Cube")
+            update.add_argument("--schema_name", dest="schema_name", help="schema_name")
+            update.add_argument("--table_name", dest="table_name", help="Table name")
+            update.add_argument("--grouping_column", dest="grouping_column", help="Grouping Column")
+            update.add_argument("--destination_id", dest="destination_id", help="Data Source id acting as destiantion")
+            update.add_argument("--query", dest="query", help="Query")
+
+        elif nezha_entity == "partitions":
+            update.add_argument("id", help="Numeric id of the Partition")
+            update.add_argument("--cost", dest="cost", help="Cost for partitions")
+            update.add_argument("--description", dest="description", help="Description")
+            update.add_argument("--name", dest="name", help="Name of the Partition")
+            update.add_argument("--schema_name", dest="schema_name", help="schema_name")
+            update.add_argument("--table_name", dest="table_name", help="Table name")
+            update.add_argument("--destination_id", dest="destination_id", help="Data Source id acting as destiantion")
+            update.add_argument("--query", dest="query", help="Query")
+
+        elif nezha_entity == "default_datasource":
+            update.add_argument("--id", help="Numeric id of the Default DataSource")
```

Comment:
Why is this `--id` and not `id`? If it has some other function than other `id`s then please update the help message to reflect that.